enhancements/update/do-not-block-on-degraded-true-clusteroperators.md

```diff
@@ -0,0 +1,211 @@
+If any turn up, we can mitigate by [declaring conditional update risks](targeted-update-edge-blocking.md) using the existing `cluster_operator_conditions{condition="Degraded"}` PromQL metric, while teaching the relevant operators to set `Available=False` and/or without their `versions` bumps until the issue that needs to block further ClusterVersion update progress has been resolved.
+
+How will security be reviewed and by whom?
+Unclear.  Feedback welcome.
+
+How will UX be reviewed and by whom?
+Unclear.  Feedback welcome.
+
+### Drawbacks
+
+As discussed in [the *Risks* section](#risks-and-mitigations), the main drawback is changing behavior that we've had in place for many years.
+But we do not expect much customer pushback based on "hey, my update completed?!  I expected it to stick on this sad component...".
+We do expect it to reduce customer frustration when they want the update to complete, but for reasons like administrative siloes do not have the ability to recover a component from minor degradation themselves.
+
+## Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+- What additional testing is necessary to support managed OpenShift service-based offerings?
+
+No need to outline all of the test cases, just the general strategy. Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+
+## Graduation Criteria
+
+There are no API changes proposed by this enhancement, which only affects sad-path handling, so we expect the code change to go straight to the next generally-available release, without feature gating or staged graduation.
+
+### Enhancement merge
+
+To ensure we do not surprise anyone with this shift, we are collecting acks from at least one maintainer for each ClusterOperator approving this pull request, and the [existing semantics for `status.versions[name=operator]`](/dev-guide/cluster-version-operator/dev/clusteroperator.md#version):
+
+> There MUST be a version with the name `operator`, which is watched by the CVO to know if a cluster operator has achieved the new level.
+
+* [ ] authentication
+* [ ] baremetal
+* [ ] cloud-controller-manager
+* [ ] cloud-credential
+* [ ] cluster-api
+* [ ] cluster-autoscaler
+* [ ] config-operator
+* [ ] console
+* [ ] control-plane-machine-set
+* [ ] csi-snapshot-controller
+* [ ] dns
+* [ ] etcd
+* [ ] image-registry
+* [ ] ingress
+* [ ] insights
+* [ ] kube-apiserver
+* [ ] kube-controller-manager
+* [ ] kube-scheduler
+* [ ] kube-storage-version-migrator
+* [ ] machine-api
+* [ ] machine-approver
+* [ ] machine-config
```

Comment:
Happy to ack from the MCO side, as we should be correctly populating `status.versions`, and blocking vs. non-blocking degrades should not drastically change MCO behaviour.

Note that a lot of MCO degrades will be due to a node unable to update, which means that a new incoming update will likely be similarly blocked until the MCO degrade is cleared.

Reply:
Thanks!  Ack recorded with 111c8fe4 -> d4a4682d , and I'll mark this thread resolved.